### ISSUE TITLE
Cannot locate specified Dockerfile: Dockerfile

### ISSUE THREAD

**Issue by ylogx** [kubernetes/kompose#809]
When I run `kompose up`, I get the following error:
```
FATA Error while deploying application: k.Transform failed: Unable to build Docker image for service business-api: Unable to build image. For more output, use -v or --verbose when converting.: API error (500): {"message":"Cannot locate specified Dockerfile: Dockerfile"}
```

My folder structure is: 
```
repo 
  package
    business-api
        Dockerfile
```

My docker-compose.yaml is:
```yaml
version: '2'

services:
  business-api:
    restart: always
    build: ./package/business_api
    image: index.docker.io/company/business-api
    volumes:
      - ./:/usr/src/app:ro
      - ./data:/root/data:ro
    depends_on:
      - some-other-service
```

**Comment by cdrage** [comment]
Hmm, may be a bug with the pathing, you may have to manually specify by doing: `./package/business_api/Dockerfile` under build.

Try that with the `--verbose` output and let post the debug information here!

**Comment by shubhamchaudhary** [comment]
Added full path in build -> dockerfile:

```
    build:
      context: ./package/business_api
      dockerfile: ./package/business_api/Dockerfile
```

Output of `sudo kompose up -v 1000`:

```
DEBU Docker Compose version: 2
DEBU Opening compose files: docker-compose.yml
DEBU [0/0] [webapp_npm]: EventType: 31
DEBU [0/0] [webapp_build]: EventType: 31
DEBU [0/0] [webapp_config]: EventType: 31
DEBU [0/0] [webapp_node_modules]: EventType: 31
DEBU [0/1] [nginx]: Adding
DEBU [0/5] [webapp]: Adding
DEBU [0/5] [business-api]: Adding
DEBU [0/6] [default]: EventType: 32
DEBU [0/6] [default]: EventType: 32
DEBU [0/6] [default]: EventType: 32
DEBU [0/6] [default]: EventType: 32
DEBU [0/6] [default]: EventType: 32
DEBU Default network found
DEBU [0/6] [default]: EventType: 32
WARN Unsupported root level volumes key - ignoring
WARN Unsupported depends_on key - ignoring
INFO Build key detected. Attempting to build and push image 'index.docker.io/example/business-api'
DEBU Compose file dir: /home/chaudhary/code/work/recs
INFO Building image 'index.docker.io/example/business-api' from directory 'business_api'
DEBU Created temporary file /tmp/kompose-image-build-853848623 for Docker image tarballing
FATA Error while deploying application: k.Transform failed: Unable to build Docker image for service business-api: Unable to build image. For more output, use -v or --verbose when converting.: API error (500): {"message":"Cannot locate specified Dockerfile: Dockerfile"}
```

This is the same output that I get when build->dockerfile is not specified as shown in the original issue at the top.

**Comment by cdrage** [comment]
@shubhamchaudhary I'll investigate! Why do you add `1000` to the `sudo kompose up -v` command?

**Comment by shubhamchaudhary** [comment]
@cdrage I thought it is `--verbose` parameterized (which it isn't now that I check --help). I added 1000 as the log level value in my head.

**Comment by hangyan** [comment]
fixed by #1212
